### PR TITLE
Partial fix for PLT-203

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

File: web/web.go [hunk 1]
```diff
@@ -145,14 +145,8 @@ func root(c *api.Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if len(c.Session.UserId) == 0 {
```
Comment (crspeller) [inline web/web.go]: Why remove this functionality? A logged in user going to root probably doesn't want to see the signup page and will be confused.

Comment (coreyhulen) [inline web/web.go]: yes, we're getting ride of all the redirect stuff.

Comment (crspeller) [inline web/web.go]: So you plan on restoring this feature in another way? What's wrong with redirects?

Comment (coreyhulen) [inline web/web.go]: When we fix multi team login we'll do something here.

Comment (crspeller) [inline web/web.go]: Added to PLT-44

**Comment by crspeller** [comment]
+1 Looks good.

**Comment by jwilander** [comment]
+1

**Pull request merged**